package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@asl/pages",
-  "version": "30.2.7",
+  "version": "30.2.4",
```

Comment:
Will this work with newer versions already published to npm - should we instead be releasing a newer version with the reverted code?